lib/matplotlib/axes/_base.py

```diff
@@ -2331,6 +2331,23 @@ def add_child_axes(self, ax):
     def add_collection(self, collection, autolim=True):
         """
         Add a `.Collection` to the Axes; return the collection.
+
+        Parameters
+        ----------
+        collection : `.Collection`
+            The collection to add.
+        autolim : bool
+            Whether to update data and view limits.
+
+            .. versionchanged:: 3.11
+
+               This now also updates the view limits, making explicit
+               calls to `~.Axes.autoscale_view` unnecessary.
+
+            As an implementation detail, the value "_datalim_only" is
```

Comment:
I'm a bit confused as to why this is in the public docstring, and not just in a code comment below?

Reply:
Ah I see now, sorry for the noise.